### PR TITLE
update documentation

### PR DESCRIPTION
**Pull request by YuvHayun**
## Status
Ready/In Progress/In Hold (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-10606

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: docs/documentation/images_in_documentation_files.md [hunk 1]
```diff
@@ -7,82 +7,26 @@ title: Images in Documentation Files
 
 Images in documentation markdown files are divided into 2 different types:
 - Images that appear in integration/script/playbook readmes. These images only appear in https://xsoar.pan.dev/. They do not appear in the Cortex XSOAR/XSIAM product UI.
-- Images that appear in pack readmes and integration description files. These images appear in both https://xsoar.pan.dev/ and in the Cortex XSOAR/XSIAM product UI. 
+- Images that appear in pack readmes and integration description files. These images appear in both https://xsoar.pan.dev/ and in the Cortex XSOAR/XSIAM product UI.
 
-## Integration/Script/Playbook Readme Images
+## Rules for uploading images to markdown files:
```
Comment (JudahSchwartz) [inline docs/documentation/images_in_documentation_files.md:12]:
```suggestion
## Instructions for uploading images to markdown files:

```

File: docs/documentation/images_in_documentation_files.md [hunk 2]
```diff
@@ -7,82 +7,26 @@ title: Images in Documentation Files
...
 Images in documentation markdown files are divided into 2 different types:
 - Images that appear in integration/script/playbook readmes. These images only appear in https://xsoar.pan.dev/. They do not appear in the Cortex XSOAR/XSIAM product UI.
-- Images that appear in pack readmes and integration description files. These images appear in both https://xsoar.pan.dev/ and in the Cortex XSOAR/XSIAM product UI. 
+- Images that appear in pack readmes and integration description files. These images appear in both https://xsoar.pan.dev/ and in the Cortex XSOAR/XSIAM product UI.
 
-## Integration/Script/Playbook Readme Images
+## Rules for uploading images to markdown files:
 
-When creating markdown `README` documents for playbooks, integrations, or scripts that appear in https://xsoar.pan.dev/ only, you can use a relative or absolute URL.
+When creating markdown documentation files, you should use a relative path to the `doc_files` folder located in the pack's root path.
```
Comment (JudahSchwartz) [inline docs/documentation/images_in_documentation_files.md:14]: give example

**Review by JudahSchwartz — COMMENTED** [review]
amazing, tag doc-review

File: docs/documentation/images_in_documentation_files.md (hunk 2 above)
Comment (YuvHayun) [reply docs/documentation/images_in_documentation_files.md:15]: There's an example several lines below in line 20.

File: docs/documentation/images_in_documentation_files.md [hunk 3]
```diff
@@ -7,82 +7,27 @@ title: Images in Documentation Files
...
-* URL after redirection (also valid): https://raw.githubusercontent.com/demisto/content/master/Packs/AutoFocus/doc_files/AutoFocusPolling.png
+### Absolute URLs
 
-:::note
-To keep our main Content repo small we limit images to 2MB. For larger images, follow the instructions for [Videos](#videos) on how to store large media files in our [content-assets](https://github.com/demisto/content-assets) repository. 
-:::
\ No newline at end of file
+Note that this case is only relevant when dealing with larger file types (such as gif and mp4 files).
+In the case, the file should be uploaded to `content-assets` repo.
+For more information, please refer to [large files absolute image urls](../documentation/readme_file#Large-Files).
```
Comment (ShirleyDenkberg) [inline docs/documentation/images_in_documentation_files.md:33]:
```suggestion
For more information, refer to [large files absolute image urls](../documentation/readme_file#Large-Files).

```

File: docs/documentation/images_in_documentation_files.md [hunk 4]
```diff
@@ -7,82 +7,27 @@ title: Images in Documentation Files
...
-* URL to `master` branch: https://github.com/demisto/content/raw/master/Packs/AutoFocus/doc_files/AutoFocusPolling.png
-* URL after redirection (also valid): https://raw.githubusercontent.com/demisto/content/master/Packs/AutoFocus/doc_files/AutoFocusPolling.png
+### Absolute URLs
 
-:::note
-To keep our main Content repo small we limit images to 2MB. For larger images, follow the instructions for [Videos](#videos) on how to store large media files in our [content-assets](https://github.com/demisto/content-assets) repository. 
-:::
\ No newline at end of file
+Note that this case is only relevant when dealing with larger file types (such as gif and mp4 files).
+In the case, the file should be uploaded to `content-assets` repo.
```
Comment (ShirleyDenkberg) [inline docs/documentation/images_in_documentation_files.md:32]:
```suggestion
Note that using absolute URLs is only relevant when dealing with larger file types (such as gif and mp4 files).
In that case, the file should be uploaded to `content-assets` repo.

```

**Comment by ShirleyDenkberg** [comment]
@JudahSchwartz Doc review completed.

**Review by JudahSchwartz — APPROVED** [review]

**Pull request merged**